protolite-well-known-types/protolite-well-known-types.gradle

```diff
@@ -26,7 +26,7 @@ firebaseLibrary {
 
 protobuf {
     protoc {
-        artifact = "com.google.protobuf:protoc:$protocVersion"
+        artifact = libs.protoc.get().toString()
```

Comment:
Just to be sure, we **do not** need to release this library, right?

Reply:
Actually, I don't need to block the merge on this. I'll approve the change but left the comment here for you to resolve whenever you have time. Thanks!